.gitignore

```diff
@@ -1,4 +1,3 @@
 node_modules
 .DS_Store
 .idea
-tck/package-lock.json
```

Comment:
Why do we want to ignore this?
Keeping the `package-lock.json` makes sure that the dependency resolution is consistent and running the same code always gives back the same results.

Reply:
Not completely sure, but I think it was because the dependency to the sdk in the `package.json` is now directly to the package (tgz file), and the sha for this keeps changing and causing issues. I certainly got those problems in this PR in CI on the first builds. I think it used to link to the sdk directory, but now uses the package.

Reply:
This change was in #77

Reply:
Ahh got it.
Yeah, unfortunately we won't be able to keep the `package-lock.json` if the hash of the sdk is always dynamically changing with each build.

There is only one solution what comes to my mind:
We could use [husky](https://github.com/typicode/husky) to create a pre-commit hook which always packages the sdk before the commit and uses the new hash to monkey patches the `package-lock.json`.
But it would the actual commit would generate a new outgoing change and it slows things down, so I am not sure that this is the right answer to our problem. It feels very dirty.... 😬 
Apparently others had this problem as well: https://github.com/npm/cli/issues/517 🤷‍♂️